DirectEmployers/MyJobs
# [PD-3467] clean up map components

## What's this PR do?
PR cleans up the map components

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
start inside of any analytics app or view the code

## How should this be manually tested?
Pull down the repository. Go into Analytics and make sure the maps are working correctly and you should also view the code changes

## Any background context you want to provide?
